Permissioned DEX XLS.md

```diff
@@ -0,0 +1,467 @@
+* **Open Offer/Payment** or **Unpermissioned Offer/Payment**: An offer/cross-currency payment that is able to trade on the open DEX or potentially in permissioned DEXes, but doesn't have any restrictions itself.
+* **Valid Domain Offer/Payment**: An offer/cross-currency payment that satisfies the rules of a domain (i.e. the account is a domain member, and the tokens in the offer are accepted). This can be a permissioned _or_ open offer.
+
+### 1.3. Basic Flow
+
+#### 1.3.1. Initial setup
+* Owen, a domain owner, creates his domain with a set of KYC credentials and allowed tokens (including a certain USD and EUR).
+* Tracy, a trader, is operating in Owen's regulatory environment and has strict regulatory requirements - she cannot receive liquidity from non-KYCed accounts. She has one of Owen's accepted KYC credentials, and will only be placing permissioned offers.
+* Marko, a market maker, wants to arbitrage offers in Owen's domain, as there is often a significant price difference inside and outside. He obtains one of the KYC credentials that Owen's domain accepts. However, he will not be placing permissioned offers, only open offers, to access as much liquidity as possible.
+
+#### 1.3.2. Trading Scenario 1
+
+* Tracy places a permissioned offer on the XRP-USD orderbook. There are no other offers on that orderbook that are a part of the domain.
+* Marko notices that there is a significant price difference between Tracy's offer and the rest of the orderbook. He now submits an open offer to cross and fill Tracy's offer.
+
+#### 1.3.3. Trading Scenario 2
+* Marko has placed open offers on the XRP-EUR orderbook.
+* Tracy places an offer on the XRP-EUR orderbook that crosses one of Marko's offers. Her offer is filled by his.
+
+### 1.4. How Domains Work
+
+A permissioned offer will only be filled by valid domain offers.
+
+An open offer can be filled by any offer on the open DEX, but can _also_ be filled by any permissioned offer, if the open offer is a valid domain offer.
+
+The fact that traders don't need to place a permissioned offer in order to fill offers in that domain enables arbitrage and mixing of liquidity much more easily, as traders do not have to place multiple offers in multiple domains. It also prevents the creation of walled gardens of liquidity.
+
+## 2. On-Ledger Object: `PermissionedDomain`
+
+This object represents a DEX domain.
+
+### 2.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`LedgerIndex`| ✔️|`string`|`Hash256`|The unique ID of the ledger object.|
+|`LedgerEntryType`| ✔️|`string`|`UInt16`|The ledger object's type (`PermissionedDomain`).|
+|`Owner`| ✔️|`string`|`AccountID`|The account that controls the settings of the domain.|
+|`Sequence`|✔️|`number`|`UInt32`|The `Sequence` value of the `OfferCreate` transaction that created this offer. Used in combination with the `Account` to identify this offer.|
```

Comment:
Description is for `OfferCreate`. Is it right?

Reply:
Copy paste typo - fixed